File: src/main/java/br/com/correios/enderecador/cep/CepInvalidoException.java
package br.com.correios.enderecador.cep;

import br.com.correios.enderecador.dao.DaoException;

public class CepInvalidoException extends DaoException {
    public CepInvalidoException() {
    }

    public CepInvalidoException(String msg) {
        super(msg);
    }

    public CepInvalidoException(String msg, Throwable t) {
        super(msg, t);
    }
}

File: src/main/kotlin/br/com/correios/enderecador/cep/CepInvalidoException.kt
package br.com.correios.enderecador.cep

import br.com.correios.enderecador.dao.DaoException

class CepInvalidoException : DaoException {
    constructor(msg: String?) : super(msg)
    constructor(msg: String?, t: Throwable?) : super(msg, t)
}
